functions/Update-DbaInstance.ps1

```diff
@@ -0,0 +1,244 @@
+        CU7 - will update all existing SQL Server versions to CU7 of current Service Pack installed
+
+    .PARAMETER MajorVersion
+        Designed to work in conjunction with -Latest to only update specific version(s) of SQL Server. Syntax: SQL20XX or simply 20XX.
+
+    .PARAMETER Latest
+        Install latest SP and CU known to the module. More details can be found by running Test-DbaBuild -SqlInstance YourSqlServer
+
+        Note: `-Latest -Type CumulativeUpdate` will try to install CU from the latest SP available. Make sure to upgrate servers to proper SP beforehand.
+
+    .PARAMETER RepositoryPath
+        Path to the folder(s) with SQL Server patches downloaded. It will be scanned recursively for available patches.
+        Path should be available from both server with SQL Server installation and client that runs the command.
+        All file names should match the pattern used by Microsoft: SQLServer####*-KB###-*x##*.exe
+        If a file is missing in the repository, the installation will fail.
+        Consider setting the following configuration if you want to omit this parameter: `Set-DbatoolsConfig -Name Path.SQLServerUpdates -Value '\\path\to\updates'`
+
+    .PARAMETER Restart
+        Restart automatically if required.
+
+    .PARAMETER WhatIf
+        Shows what would happen if the command were to run. No actions are actually performed.
+
+    .PARAMETER Confirm
+        Prompts you for confirmation before executing any changing operations within the command.
+
+    .PARAMETER EnableException
+        By default, when something goes wrong we try to catch it, interpret it and give you a friendly warning message.
+        This avoids overwhelming you with "sea of red" exceptions, but is inconvenient because it basically disables advanced scripting.
+        Using this switch turns this "nice by default" feature off and enables you to catch exceptions with your own try/catch.
+
+    .NOTES
+        Tags: Install, Patching, SP, CU, Instance
+        Author: Kirill Kravtsov (@nvarscar) https://nvarscar.wordpress.com/
+
+        Website: https://dbatools.io
+        Copyright: (c) 2018 by dbatools, licensed under MIT
+        License: MIT https://opensource.org/licenses/MIT
+
+        Requires Local Admin rights on destination computer(s).
+
+    .EXAMPLE
+        PS C:\> Update-DbaInstance -ComputerName SQL1 -SqlServerVersion SP3 -RepositoryPath \\network\share
+
+        Updates all applicable SQL Server installations on SQL1 to SP3.
+        SP3 binary files will be
+
+    .EXAMPLE
+        PS C:\> Update-DbaInstance -ComputerName SQL1, SQL2 -Latest -Restart
+
+        Updates all applicable SQL Server installations on SQL1 and SQL2 with the most recent patch.
+        It will install latest ServicePack, restart the computers, install latest Cumulative Update, and finally restart the computer once again.
+
+    .EXAMPLE
+        PS C:\> Update-DbaInstance -ComputerName SQL1 -Latest -MajorVersion 2012 -Type ServicePack
```

Comment:
Can/Should Latest be implied? and thus not required?

Reply:
Yeah I wouldn't want to use `-Lastest$false` ever, perhaps the param needs to be renamed? or made into a string with a validateset?